src/main/java/net/geforcemods/securitycraft/items/TaserItem.java

```diff
@@ -73,8 +79,28 @@ else if(player.inventory.hasItemStack(oneRedstone))
 					return ActionResult.resultPass(stack);
 				}
 
-				WorldUtils.addScheduledTask(world, () -> world.addEntity(new TaserBulletEntity(world, player, powered)));
+				int range = 11;
+				Vector3d startVec = player.getEyePosition(1.0F);
+				Vector3d lookVec = player.getLook(1.0F).scale(range);
```

Comment:
The range feels short because you are scaling by `range` and not `range * range` here. 
```suggestion
				Vector3d lookVec = player.getLook(1.0F).scale(range * range);
```

Reply:
1. Both the examples that use `ProjectileHelper.rayTraceEntities` use the distance and not the square of the distance in that place
2. For me nothing changes if I change the parameter from `range` to `range * range`. 
Should I still apply the change?

Reply:
Hm...weird. In my testing earlier it was half the intended range for me with `range` and the correct range with `range * range`, but now it's the intended one with only `range`. I am confused. Leave it as is.